exercises/level9/.backstage/demoapp/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
Is the lockfile really useful here ? It bothers me a little to add 3k lines of yarn non-sense which represents roughly half the repository.